src/test/java/seedu/address/model/order/OrderTest.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model.order;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DETAILS_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static seedu.address.testutil.TypicalOrders.EMILY;
+import static seedu.address.testutil.TypicalOrders.SIMON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.OrderBuilder;
+
+public class OrderTest {
+
+    @Test
+    public void isSameOrder() {
+        // same object -> returns true
+        assertTrue(EMILY.isSameOrder(EMILY));
+
+        // null -> returns false
+        assertFalse(EMILY.isSameOrder(null));
```

Comment:
Might need to change this as well, if we decide to change the `isSameOrder` implementation